QueryBuilder.Test.Generated/QueryBuilder.Test.Generated.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net6</TargetFrameworks>
+    <TargetFrameworks>net6</TargetFrameworks>
```

Comment:
net6.0 - we've seen issues where the multi-target frameworks check fails because we don't have the full net6.0